ingestion/src/metadata/orm_profiler/profiler/runner.py

```diff
@@ -55,25 +59,76 @@ def _build_query(self, *entities, **kwargs) -> Query:
         return self._session.query(*entities, **kwargs)
 
     def _select_from_sample(self, *entities, **kwargs):
-        return self._build_query(*entities, **kwargs).select_from(self._sample)
+        """Run select statement against sample data"""
+        if kwargs.get("query_filter_"):
+            query_filter = kwargs.pop("query_filter_")
```

Comment:
sorry random question, why do we need to `pop` the keys?

Reply:
We use pop as we don't want to pass that keyword to `self._build_query` 🙂